thoth/package_analyzer/cli.py

```diff
@@ -123,15 +124,18 @@ def python(
     dry_run: bool = False,
 ):
     """Fetch digests for packages in Python ecosystem."""
+    start_time = time.time()
     python_fetcher = PythonDigestsFetcher(index_url)
     result = python_fetcher.fetch(package_name, package_version)
+    duration = start_time - time.time()
 
     print_command_result(
         click_ctx,
         result,
         analyzer=analyzer_name,
         analyzer_version=analyzer_version,
         output=output or "-",
+        duration=duration,
```

Comment:
The duration you pass here is `float`, the function expects `int` based on your PR in https://github.com/thoth-station/analyzer/pull/186. You can pass directly float and explicitly cast to int in https://github.com/thoth-station/analyzer/pull/186

Reply:
Done!

Reply:
Awesome! :+1: